images/fluentd/tests/main.tf

```diff
@@ -11,4 +11,8 @@ variable "digest" {
 data "oci_exec_test" "test" {
   digest = var.digest
   script = "${path.module}/01-test.sh"
+  env {
+    name  = "MODULE_PATH"
+    value = path.module
+  }
```

Comment:
I think you want `working_dir = path.module` instead, then `pwd` in the scripts will be more predictable

Reply:
`pwd` will be the `path.module` 😁

Reply:
:chefs-kiss"